src/content/docs/release-notes/docs-release-notes/docs-6-14-2024.mdx

```diff
@@ -0,0 +1,85 @@
+---
+subject: "Docs"
+releaseDate: '2024-06-07'
+version: 'version: June 7-13, 2024'
+---
+
+### New docs
+
+* [NR24-01 security bulletin](/docs/security/new-relic-security/security-bulletins/security-bulletin-nr24-01) addresses the recent Fluent Bit security vulnerability.
+* A new [Cacti integration](/docs/infrastructure/host-integrations/host-integrations-list/cacti-integration) gives you information on how to see your RRD data in New Relic.
+
+### Major changes
+
+* We've moved away from the `applied intelligence` name, because it's confusing when compared to our other AI efforts. We completed a massive sweep to remove nearly all mention of "applied intelligence" from our docs.
+* Deleted several old, outdated alerts docs.
+* A rewrite of [change vulnerability status](/docs/vulnerability-management/change-vulnerability-status) makes it easier for you to enable this feature.
+
+### Minor changes
+
+* Removed the notifications section from [CodeStream settings](/docs/codestream/codestream-settings/account-settings).
+* Added a new OpenTelemetry [redis instance](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-resources) section to clarify that you need to manually include redis attributes for the collector.
```

Comment:
```suggestion
* Added a new OpenTelemetry [redis instance](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-resources) section to clarify that you need to manually include Redis attributes for the collector.
```

maybe?

Reply:
Redis x2 but i'm not sure!

Reply:
In the doc update itself, it was mostly lowercase....